unittest/casadi-basic.cpp

```diff
@@ -52,8 +49,8 @@ BOOST_AUTO_TEST_CASE(test_example)
   {
     for (Eigen::Index j = 0; j < A.cols(); ++j)
     {
-      A(i, j) = 10 * i + j;
-      B(i, j) = -10 * i - j;
+      A(i, j) = 10. * static_cast<double>(i) + static_cast<double>(j);
```

Comment:
Did you have some compiler warnings here?

Reply:
Yes

Reply:
Which compiler are you using?

Reply:
g++ (Ubuntu 8.3.0-6ubuntu1) 8.3.0